blathering-barnacles/Bangazon-3
# Completed ticket #3 . Payment Types

## Link to Ticket
Closes #3 [https://github.com/blathering-barnacles/Bangazon-3/issues/3]()

## Description of Proposed Changes
Set up ability to view, post, edit or delete the payment types and each individual payment type. 
- 

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout <rl-tick3_paymenttype>
```
Go to http://localhost:8000/api/v1/paymenttypes
You should see a list of payment types and be able to click on an indiviudual URL
You should also be able to add a new payment type from here
Go to http://localhost:8000/api/v1/paymenttypes/1
You should be able to view, edit or delete an individual payment type


## Impacted Areas in Application

List general components of the application that this PR will affect:
Payment Type List view and Detail view
*
